client/src/components/payments/checkoutForm.js

```diff
@@ -103,8 +108,7 @@ class _CheckoutForm extends Component {
       <Fragment>
         <CardContent className={classes.content}>
           <Typography
-            className={"MuiTypography--heading"}
-            style={ {textAlign: "center", margin: "15px 0 -10px"}}
+            className={`MuiTypography--heading ${classes.paymentTypo}`}
```

Comment:
Just curious as to why MuiTypography--heading is in the className

Reply:
because I wanted to alter it